dff/script/conditions/std_conditions.py

```diff
@@ -27,7 +27,9 @@ def exact_match(match: Any, *args, **kwargs) -> Callable[[Context, Actor, Any, A
 
     def exact_match_condition_handler(ctx: Context, actor: Actor, *args, **kwargs) -> bool:
         request = ctx.last_request
-        return match == request
+        if request is None:
+            return False
+        return match == request.text
```

Comment:
I think text match is a good option for  `regexp`. We can remove `exact_match` at all. What do you think?

Reply:
I think just remove `exact_match`

Reply:
I don't think this is a good idea. While `exact_match` may not be useful in practice, it is still a very good function to showcase the framework (which is why most of the examples use it).